umi_alignment/call_duplex_consensus.cwl

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: 'call duplex consensus'
+baseCommand: ["/usr/bin/java", "-Xmx4g", "-jar", "/opt/fgbio-0.5.0.jar", "CallDuplexConsensusReads"]
+arguments:
+    ["--error-rate-pre-umi", "45", "--error-rate-post-umi", "30", "--min-input-base-quality", "30",
```

Comment:
Is the `--min-reads` option also required for this command? I don't see those parameters nor are defaults documented here:
http://fulcrumgenomics.github.io/fgbio/tools/latest/CallDuplexConsensusReads.html

Reply:
Never mind, I think I understand the docs a bit more now. The default would be `1 1 1`